xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxCC.cpp

```diff
@@ -11,9 +11,80 @@
 #include "DVDDemuxUtils.h"
 #include "cores/VideoPlayer/DVDCodecs/Overlay/contrib/cc_decoder708.h"
 #include "cores/VideoPlayer/Interface/TimingConstants.h"
+#include "utils/ColorUtils.h"
+#include "utils/StringUtils.h"
 
 #include <algorithm>
 
+namespace
+{
+/*!
+ * \brief Given the color code from the close caption decoder, returns the corresponding
+ * color in rgb format (striping the alpha channel from argb)
+ * 
+ * \param[in] ccColor - a given color from the cc decoder
+ * \return the corresponding Color in rgb
+ */
+UTILS::COLOR::Color CCColorConversion(const uint8_t ccColor)
```

Comment:
```suggestion
constexpr UTILS::COLOR::Color CCColorConversion(const uint8_t ccColor)
```

Reply:
can you explain why is need to add constexpr? its not already static?

Reply:
also to know if we need to add this as code guideline rule